docs/Introduction/architecture-overview/off-chain-reporting.md

```diff
@@ -4,59 +4,42 @@ section: ethereum
 date: Last Modified
 title: "Off-Chain Reporting"
 permalink: "docs/off-chain-reporting/"
-metadata: 
-  image: 
+metadata:
+  image:
     0: "/files/fb73165-cl.png"
 ---
-Off-Chain Reporting (OCR) is the next Oracle network upgrade replacing the [`FluxAggregator` model](../architecture-decentralized-model/). It represents a significant step towards increasing the decentralization and scalability of Chainlink networks.
 
-See the <a href="https://uploads-ssl.webflow.com/5f6b7190899f41fb70882d08/603651a1101106649eef6a53_chainlink-ocr-protocol-paper-02-24-20.pdf" target="_blank">OCR protocol paper</a> for a technical deep dive.
+Off-Chain Reporting (OCR) is a significant step towards increasing the decentralization and scalability of Chainlink networks.
```

Comment:
Small detail but should we link to more info here?

Reply:
I found a more recent whitepaper. Good call! https://research.chain.link/ocr.pdf